js/whitebit.js

```diff
@@ -503,42 +572,65 @@ module.exports = class whitebit extends Exchange {
         //         "isBuyerMaker":false
         //     }
         //
+        // orderTrades (v4Private)
+        //       {
+        //          "time": 1593342324.613711,      // Timestamp of executed order
+        //          "fee": "0.00000419198",         // fee that you pay
+        //          "price": "0.00000701",          // price
+        //          "amount": "598",                // amount in stock
+        //          "id": 149156519,                // trade id
+        //          "dealOrderId": 3134995325,      // completed order Id
+        //          "clientOrderId": "customId11",  // custom order id; "clientOrderId": "" - if not specified.
+        //          "role": 2,                      // Role - 1 - maker, 2 - taker
+        //          "deal": "0.00419198"            // amount in money
+        //       }
         let timestamp = this.safeValue (trade, 'time');
         if (typeof timestamp === 'string') {
             timestamp = this.parse8601 (timestamp);
         } else {
             timestamp = parseInt (timestamp * 1000);
         }
+        const orderId = this.safeString (trade, 'dealOrderId');
         const priceString = this.safeString (trade, 'price');
-        const amountString = this.safeString2 (trade, 'amount', 'volume');
+        let amountString = this.safeString (trade, 'deal');
```

Comment:
Need a bit of your comment on these lines. According to our definitions, cost = amount * price, does this condition hold if we use `'deal'` and `'amount'` here interchangeably?

Reply:
@kroitor  Actually I don't think so, since 'deal' is the amount in the money currency (quoteCurrency). So, correct me if I'm wrong, the 'deal' is already the cost since price * amountInBaseCurrency = quoteCurrency.

I'm going to change it accordingly

Reply:
Yes, basically, we should figure what is what, (`cost` = `amount` * `price` should hold), and we fill as much info from the exchange as we can (including cost). If `cost` is not defined by the exchange – we leave it undefined, and it will be calculated from `amount` and `price` in `safeTrade`. So, if a field is defined by the exchange – we fill it in to prefer the info from the exchange. If it's missing – then it's undefined.